redux-excersice/src/components/FormComponent.js

```diff
@@ -1,8 +1,66 @@
-import React from "react";
+import React, { useCallback, useState } from "react";
+import { object, string, ref } from 'yup';
 
 export default function FormComponent() {
```

Comment:
What happends if an email that comes from the url is not valid?
How can we validate them?

Reply:
yep, We will in other ticket